ktoml-core/src/commonTest/kotlin/com/akuleshov7/ktoml/parsers/ValueParserTest.kt

```diff
@@ -1,5 +1,6 @@
 package com.akuleshov7.ktoml.parsers
 
+import com.akuleshov7.ktoml.TomlConfig
```

Comment:
Could you please also add a "decoder" test in the decoder test directory?
And fix existing, if it is possible for you 🙏